dmriprep/utils/images.py

```diff
@@ -0,0 +1,113 @@
+import numpy as np
```

Comment:
This new file makes a lot of sense, especially if we can come up with unit tests for all these helper functions.

Could you send a separate PR to split the currently existing functions (extract_b0 and rescale_b0) out to this file so they don't show up in the diff?

Reply:
Yes.